src/test/test-seccomp.c

```diff
@@ -30,7 +30,7 @@
 #include "virt.h"
 
 /* __NR_socket may be invalid due to libseccomp */
-#if !defined(__NR_socket) || __NR_socket < 0 || defined(__i386__) || defined(__s390x__) || defined(__s390__)
+#if !defined(__NR_socket) || __NR_socket < 0 || defined(__i386__) || defined(__s390x__) || defined(__s390__) || defined(__powerpc__)
 /* On these archs, socket() is implemented via the socketcall() syscall multiplexer,
  * and we can't restrict it hence via seccomp. */
```

Comment:
Hmm, this comment sounds incompatible with the comment in `seccomp_restrict_address_families()`:
```
/* These we know we support (i.e. are the ones that do not use socketcall()) */
```
Is it not necessary to revert the change for `seccomp_restrict_address_families()` in da1921a5c396547261c8c7fcd94173346eb3b718?

Reply:
You're right, a revert is in order. Updated in place.

Reply:
This passes on ppc64 with libseccomp-2.5.0-3.fc33.ppc64le.